contracts/hts-precompile/ERC20Contract.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+
+contract ERC20Contract {
```

Comment:
To Danno's point this should be under examples
Place it under `contracts/hts-precompile/examples` in an `erc20` dir